lib/facil/http/http1.c

```diff
@@ -342,6 +345,22 @@ static int http1_http2websocket_server(http_s *h, websocket_settings_s *args) {
     goto bad_request;
   stmp = fiobj_obj2cstr(tmp);
 
+  FIOBJ tmp_protocol = fiobj_hash_get2(h->headers, sec_protocol);
+  if (tmp_protocol) {
+    char *protocol_str = fiobj_obj2cstr(tmp_protocol).data;
+
+    if (strlen(protocol_str)) {
+      printf("HALLO\n");
```

Comment:
And this is some debugging line? if required, it is better to use `FIO_LOG_DEBUG` with a proper message.